AdobeStockImageAdminUi/Test/Mftf/Section/AdobeStockSection.xml

```diff
@@ -23,5 +23,7 @@
         <element name="savePreview" type="block" selector="//button[@class='action-secondary']/span[contains(.,'Save Preview')]"/>
         <element name="imagePreview" type="block" selector="//div[@class='masonry-image-preview']//img"/>
         <element name="imagePreviewNavigation" type="block" selector="//div[@class='masonry-image-preview']//div[@class='action-buttons']/button[@class='action-{{type}}']" parameterized="true"/>
+        <element name="imageAttribute" type="block" selector="//*[@id='adobe-stock-images-search-modal']//div[@class='attributes']//span[text()='{{type}}']/parent::div//div[@class='value']//span" parameterized="true"/>
```

Comment:
Just a question, why do you prefer XPath selectors instead of CSS selectors?

Reply:
Hi @hws47a thank you for your review, for me it's easier to debug or find the element. for example to find element i'm use this` $x("//small/parent::*[@class='filecnt selected']//img")` and i think for perfomance it's better to use xpath,  but as devdoc say's this is not good practice, so i agree with you and change this according to devdocs.
![DeepinScreenshot_select-area_20190827093531](https://user-images.githubusercontent.com/51681547/63746866-725be300-c8ae-11e9-99fb-769ea12d6301.png)